training_wheels/tests/data/test_dataset_analysis.py

```diff
@@ -0,0 +1,35 @@
+from training_wheels.data.dataset_analysis import get_label_imbalance_array, get_max_label_imbalance
+
+import numpy as np
+import pytest
+
+
+def test_label_imbalance_1():
+    expected = np.asarray([
+        [0, 0.375, 0.5],
+        [-0.375, 0, 0.125],
+        [-0.5, -0.125, 0]
+    ])
+    actual = get_label_imbalance_array([3,2,1,2,1,1,1,1])
+    for i in range(len(expected)):
+        for j in range(len(expected[i])):
+            assert actual[i, j] == expected[i, j]
```

Comment:
Hey could you turn this into a named function? It's a bit hard to tell what's going on here